<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KafkaConsumer</string>
  </property>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>89</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>89</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>114</y>
     <width>34</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0 solod black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/folder.png</normaloff>../img/folder.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="editNotifications">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>184</y>
     <width>1181</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClean">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>89</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clean</string>
   </property>
  </widget>
  <widget class="QLabel" name="appVersion">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>678</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #999;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>4</y>
     <width>772</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background: #9bc2c4;
border: 1 solid #6aa4a8;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHideFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>114</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Hide</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0 solod black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/close.png</normaloff>../img/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="comboLocation">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>51</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>56</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>439</x>
     <y>25</y>
     <width>77</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TopicName</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboTopicName">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainServers">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>50</y>
     <width>401</width>
     <height>112</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboOffset">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>114</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Group_id </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>84</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineGroup">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>209</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxcCommit">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>auto_commit</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTopicName">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>20</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLocation">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>23</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Location </string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxEnv">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>14</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxFile">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>88</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>write to file</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxTree">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>88</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>expand json (beta)</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCase">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>88</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>case sensitive</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineFilter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>52</y>
     <width>391</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>for filtering messages: number \ state \ session_id \ etc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Environment:</string>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>btnStart</zorder>
  <zorder>btnStop</zorder>
  <zorder>btnFile</zorder>
  <zorder>labelFile</zorder>
  <zorder>editNotifications</zorder>
  <zorder>btnClean</zorder>
  <zorder>appVersion</zorder>
  <zorder>btnHideFile</zorder>
  <zorder>comboLocation</zorder>
  <zorder>label_3</zorder>
  <zorder>label_9</zorder>
  <zorder>comboTopicName</zorder>
  <zorder>plainServers</zorder>
  <zorder>comboOffset</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>lineGroup</zorder>
  <zorder>checkBoxcCommit</zorder>
  <zorder>lineTopicName</zorder>
  <zorder>checkBoxLocation</zorder>
  <zorder>comboBoxEnv</zorder>
  <zorder>checkBoxFile</zorder>
  <zorder>checkBoxTree</zorder>
  <zorder>checkBoxCase</zorder>
  <zorder>lineFilter</zorder>
  <zorder>label_6</zorder>
 </widget>
 <tabstops>
  <tabstop>lineFilter</tabstop>
  <tabstop>checkBoxCase</tabstop>
  <tabstop>btnStart</tabstop>
  <tabstop>btnStop</tabstop>
  <tabstop>btnFile</tabstop>
  <tabstop>lineTopicName</tabstop>
  <tabstop>lineGroup</tabstop>
  <tabstop>comboOffset</tabstop>
  <tabstop>checkBoxcCommit</tabstop>
  <tabstop>plainServers</tabstop>
  <tabstop>editNotifications</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
